docs/ProductSummaryCustom.md

```diff
@@ -0,0 +1,110 @@
+📢 Use this project, [contribute](https://github.com/vtex-apps/product-summary/blob/master/docs/ProductSummaryBuyButton.md) to it or open issues to help evolve it using [Store Discussion](https://github.com/vtex-apps/store-discussion).
+
+# Product Summary Custom
+
+Product Summary Custom is a block exported by the Product Summary app responsible for rendering the Product Summary.
+
+![](https://user-images.githubusercontent.com/40380674/96649443-7d21d480-1307-11eb-9100-534fa9e70ca6.png))
+
+## Configuration
+
+1. Add the [Product Summary app](https://vtex.io/docs/components/content-blocks/vtex.product-summary/) to your theme's dependencies on the `manifest.json`, for example:
+
+```json
+  dependencies: {
+    "vtex.product-summary": "2.x"
+  }
+```
+
+2. Declare the `product-summary.shelf` block.
```

Comment:
If the doc title is Product Summary Custom, why are the instructions related to the Product Summary Shelf? They seem to be the *same* to me, but I don't know if I have understood it correctly!

Reply:
Also, the Product Summary Custom is not added to the main table of blocks in the Product Summary app documentation. Shouldn't it be there?

Reply:
Yes, they are the same. In this project, the component is called `ProductSummaryCustom`, but its "id" is "product-summary.shelf". I can change the title, no problem 

https://github.com/vtex-apps/product-summary/blob/master/store/interfaces.json#L9-L12